S2VX.Game.Tests/VisualTests/S2VXCursorTests/CursorReset.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Screens;
+using S2VX.Game.SongSelection;
+
+namespace S2VX.Game.Tests.VisualTests.S2VXCursorTests {
+    public class CursorReset : S2VXTestScene {
+        [Cached]
+        private ScreenStack ScreenStack { get; set; } = new();
+
+        private SongSelectionScreen SongSelectionScreen { get; set; }
+
+        [Cached]
+        private S2VXCursor Cursor { get; set; } = new S2VXCursor();
```

Comment:
Yeah, it can be `new()`. I'll change this plus any new usages going forward. I wish there was a code analysis tool to figure all these instances out for us, but I don't think the rules are enforcing something like this yet in all areas.

Reply:
Finished change. Let's not retroactively change this for our entire solution unless we're working in that file. I did a quick search and there're way too many cases of this to fix.

Reply:
Sounds good